notification-channel-config-service-impl/src/test/java/org/hypertrace/notification/config/service/NotificationChannelConfigServiceImplTest.java

```diff
@@ -38,6 +45,26 @@ void beforeEach() {
         NotificationChannelConfigServiceGrpc.newBlockingStub(mockGenericConfigService.channel());
   }
 
+  @Test
+  void testProtoChanges() throws IOException {
+    String notificationChannelFileName =
+        Objects.requireNonNull(
+                Thread.currentThread()
+                    .getContextClassLoader()
+                    .getResource("sample_notification_channel.json"))
+            .getPath();
+    String notificationChannelInJson =
+        new String(Files.readAllBytes(new File(notificationChannelFileName).toPath()));
```

Comment:
nit: lots of various shortcuts for loading JSON, e.g.
```suggestion
    String notificationChannelInJson = Resources.toString(getResource("sample_notification_channel.json"), StandardCharsets.UTF_8);
```
or
```suggestion
 String notificationChannelInJson = ConfigFactory.parseResources("sample_notification_channel.json").root().render();
```